package.json

```diff
@@ -8,65 +8,86 @@
   },
   "license": "MIT",
   "author": "Neolution",
+  "sideEffects": false,
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.modern.js",
+      "require": "./dist/index.cjs"
+    }
+  },
   "main": "dist/index.js",
+  "umd:main": "dist/index.umd.js",
+  "jsdelivr": "dist/index.umd.js",
+  "unpkg": "dist/index.umd.js",
   "module": "dist/index.modern.js",
   "source": "src/index.ts",
+  "jsnext:main": "dist/index.modern.js",
+  "types": "dist/index.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
     "build": "yarn clean-dist && yarn build-lib && yarn copy-sass",
-    "build-lib": "microbundle --jsx React.createElement --jsxFragment React.Fragment --jsxImportSource react --globals react/jsx-runtime=jsx --no-compress --tsconfig ./tsconfig.microbundle.json",
+    "build-lib": "rollup -c",
     "clean-dist": "shx rm -rf dist",
     "copy-sass": "shx cp -r styles dist/scss",
     "lint": "eslint \"**/*.{ts,tsx}\" --cache --max-warnings 0",
     "prepack": "yarn build",
     "prepare-pr": "yarn prettier . --write && yarn lint && yarn build && yarn test\"",
     "prettier-check": "prettier --check .",
-    "start": "microbundle watch --jsx React.createElement --jsxFragment React.Fragment --jsxImportSource react --globals react/jsx-runtime=jsx --no-compress --tsconfig ./tsconfig.microbundle.json",
+    "start": "rollup -c -w",
     "start-all": "concurrently \"yarn start\" \"yarn start-yalc\"",
     "start-yalc": "yarn nodemon --watch dist -x \"yarn yalc push\"",
     "test": "cross-env DEBUG_PRINT_LIMIT=100 jest",
     "tsc": "tsc"
   },
   "dependencies": {
-    "classnames": "^2.3.2"
+    "classnames": "^2.3.2",
+    "rollup-plugin-sass": "^1.12.20"
```

Comment:
please move this do dev dependencies

Reply:
yes sure, thank you, not sure how i managed to move it there 🤔